common/text/macro_definition.h

```diff
@@ -127,6 +132,8 @@ class MacroDefinition {
 
   // un-tokenized text
   DefaultTokenInfo definition_text_;
+  std::vector<TokenInfo> lexed_macro_body;
```

Comment:
Note the style to use underscores at the end of instance variables of classes (exception: `struct`s).